sass/blocks/developer/_developer.scss

```diff
@@ -1,26 +1,50 @@
 .developer {
-  &__paragraph {
+  &__svg {
+    position: absolute;
+    top: 13px;
+    left: 103px;
+  }
+
+  &__logo {
+    display: inline-block;
+    position: relative;
+
+    &:hover .developer__svg {
+      fill: $color-blue;
+    }
+
+    &:active .developer__svg {
+      opacity: 0.3;
+    }
+  }
+
+  &__text {
     display: inline-block;
+    position: relative;
+    margin-right: 40px;
+    padding-top: 16px;
+    color: $color-black;
 
     @include header(h5);
     margin-bottom: 0;
     font-weight: 400;
     text-transform: uppercase;
   }
 
-  &__logo {
-    position: relative;
-    top: 1.3rem;
-    margin-left: 5px;
-  }
+  // Tablet screen
+  @include screen-size($tablet-size) {
+    width: 33.3%;
+    padding-top: 1.5rem;
 
-  &__svg {
-    &:hover {
-      fill: $color-blue;
+    &__text {
+      margin-right: 28px;
     }
+  }
 
-    &:active {
-      opacity: 0.3;
+  // Desktop screen
+  @include screen-size($desktop-size) {
+    &__logo {
+      margin-left: 18.9rem;
```

Comment:
not sure about that kind of positioning, you element is text-aligned: center, and you add left margin to it, and it aligns it with margin. I would align the component right and add needed right margin or padding.